pkg/cmd/cluster/cluster_rename.go

```diff
@@ -8,7 +8,7 @@ import (
 // newCmdClusterRename renames a cluster.
 func newCmdClusterRename(ctx api.Context) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "rename",
+		Use:   "rename <cluster> <new_name>",
```

Comment:
Shouldn't it be `<new-name>`? Our commands (e.g. `list-providers`) use dashes.

Reply:
I took it out from the current CLI help usage (`dcos cluster rename <name> <new_name>`), I will try to find if there is a preferred notation amongst CLIs in general.

We could also call it `<name>` here.

Reply:
Yes, let's use `<name>`.

Reply:
updated